cogs/utils/subscribers.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) idoneam (2016-2020)
```

Comment:
should this file be named differently to avoid confusing its purpose with `cogs/subscribers.py`?